### PR TITLE
Sig standardization proposal

### PR DESCRIPTION
**Pull request by bennyvasquez**
One of the boundaries for folks who are new to AlmaLinux is a lack of standardization for onboarding new members to the SIGS. This PR has a lot of changes in it, but it boils down to:

* create a standardized SIG wiki page structure that each existing SIG would adopt (see the changes to the marketing SIG page for an example)
* update the process for creating new SIGs to reduce the work involved for standing up a new SIG, and remove as much of the "red tape" as possible.

### CODE REVIEW

**Comment by xsub** [comment]
@bennyvasquez We need this for Certification SIG -- perhaps we could combine efforts?

**Comment by xsub** [comment]
LGTM!

**Review by xsub — APPROVED** [review]
Approving!

File: docs/sigs/Marketing.md [hunk 1]
```diff
@@ -3,41 +3,76 @@ title: "Marketing SIG"
...
+### Help wanted
 
-## Help wanted
+Contributions take all kinds of shapes and sizes, and we welcome everyone! If you have expertise or time and would like to help us out, here are a few things that need help soon, and a place where you can find more information.
 
-* Event staffers
-* Content writers (especially for the blog and social media)
-* Website maintainers
-* Graphic designers
+* Mastodon automation - we use Sprout Social for scheduling ahead of tiem right now, but it doens't work with mastodon!
```
Comment (sboldyreva) [inline docs/sigs/Marketing.md:40]:
```suggestion
* Mastodon automation - we use Sprout Social for scheduling ahead of time right now, but it doesn't work with mastodon! 
```

Comment (bennyvasquez) [reply docs/sigs/Marketing.md:40]: Good catch!

**Pull request merged**